.github/docker/README.md

```diff
@@ -0,0 +1,35 @@
+# Content
+
+Dockerfiles and scripts placed in this directory are intended to be used as
+development process vehicles and part of continuous integration process.
+
+Images built out of those recipes may by used with Docker or podman as
+development environment.
+
+# How to build docker image
+
+To build docker image on local machine execute:
+
+```sh
+docker build --build-arg https_proxy=http://proxy.com:port --build-arg http_proxy=http://proxy.com:port -t ur:ubuntu-22.04 -f ./ubuntu-22.04.Dockerfile .
+```
+
+To set any build time variable (e.g., an optional ARG from docker recipe), add to the command (after `build`), e.g.:
+
+```sh
+ --build-arg SKIP_DPCPP_BUILD=1
+```
+
+# How to use docker image
+
+To run docker container (using the previously built image) execute:
+
+```sh
+docker run --network=bridge --shm-size=4G -v /your/workspace/path/:/opt/workspace:z -w /opt/workspace/ -it ur:ubuntu-22.04 /bin/bash
```

Comment:
Is `--network=bridge` necessary for using a proxy network?

Reply:
actually, I removed it, 'bridge' is the default value